cmd/mcp.go

```diff
@@ -190,4 +190,7 @@ func init() {
 	mcpRunCmd.Flags().Bool("stdio", true, "Run the MCP server in Stdio mode")
 	mcpRunCmd.Flags().Bool("sse", false, "Run the MCP server in SSE mode")
 	mcpRunCmd.MarkFlagsMutuallyExclusive("stdio", "sse")
+	
+	manualInstallInfo := "\n\nManual Installation:\nIf you need to manually configure an MCP client, use this command:\n\n  /home/ubuntu/repos/cli/agentuity mcp run --stdio"
+	mcpCmd.Long = mcpCmd.Long + manualInstallInfo
```

Comment:
just move this into long example in the mcpCmd

don't worry about getting correct path. just assume agentuity is the path